src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/ReportCommonMethods.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.reports;
+
+import com.microsoft.azure.servicebus.primitives.StringUtil;
+import uk.gov.hmcts.ecm.common.model.ccd.items.DateListedTypeItem;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.OLD_DATE_TIME_PATTERN;
+import static uk.gov.hmcts.ethos.replacement.docmosis.reports.memberdays.MemberDaysReport.OLD_DATE_TIME_PATTERN3;
+
+public class ReportCommonMethods {
+
+    private ReportCommonMethods() {
+    }
+
+    public static String getHearingDuration(DateListedTypeItem c) {
```

Comment:
Duration in what? Minutes? Hours? Either you code has to be self-documenting or you provide a javadoc

Reply:
refactored/renamed method name